.github/pull_request_template.md

```diff
@@ -4,6 +4,6 @@
 
 ## Checklist before requesting a review
```

Comment:
A general question is whether this can be made part of the template for the _reviewer_ in addition to it being a template for the PR-submitter